docs/submission/submitting-clinical-data.md

```diff
@@ -15,45 +16,49 @@ There is only one Clinical Submission workspace for each program. All program da
+  - When you save the Excel spreadsheet, make sure it remains a TSV file. Upon saving, you may need to confirm "do you want to keep using that format?", click "Yes".
+- If you are using a text editor, make sure you have invisible characters turned on. This way you can be sure that tabs are correctly separating each field value.
 - You do not need to remove any columns from the TSV file. For example, if the field is labeled "extended" in the dictionary and you do not have a value for that field, simply leave it blank.
-- Each file template name references the corresponding version of the dictionary (e.g argo_submission_templates_v0.7 is referencing version 0.7 of the data dictionary). Please make sure you are using the latest version of these files by referencing the latest [Data Dictionary](/dictionary). 
+- Each file template name references the corresponding version of the dictionary (e.g argo_submission_templates_v0.7 is referencing version 0.7 of the data dictionary). Please make sure you are using the latest version of these files by referencing the latest [Data Dictionary](/dictionary).
 - Check out [Clinical Data Validation Rules](clinical-data-validation-rules) for the clinical file validation rules.
 
 ### Step 2: Upload Clinical TSV Files
+
 ![Upload Files](/assets/submission/clinical-2-upload.png)
 
-1. Once your file(s) are formatted, click the **Upload Files** button and select your file(s) from the browser. Only TSV file types are supported, and the file names must begin with the data type that is being submitted.  For example, _donor_v.07-[anything].tsv_ would be a correctly formatted name for the donor file.
+1. Once your file(s) are formatted, click the **Upload Files** button and select your file(s) from the browser. Only TSV file types are supported, and the file names must begin with the data type that is being submitted. For example, _donor_v.07-[anything].tsv_ would be a correctly formatted name for the donor file.
 1. Upon uploading, if there are any errors in your file they will be displayed within the Clinical Submission workspace. The error report will also be available for download. You must fix all of the errors that are listed within your file(s) and then reupload them.
 1. Valid files will be available for review within the file preview tables. You can review the file data for each data type by using the tabs on the left of the preview area.
 
 ### Step 3: Validate Clinical Workspace
-Uploaded data must be checked for accuracy against the data in the current submission, as well as any data that has previously been submitted for your donors.
+
+Uploaded data must be checked for accuracy against the data in the current submission, as well as any data that has previously been submitted for your donors. For a list of descriptions of all the cross-file data validations that are performed, see [this](/docs/submission/clinical-data-validation-rules#cross-file-validations).
 
```

Comment:
Suggestion, (not great practice to just have the link label as "this")
Check out the [Clinical Data Validation Rules] for a list of the cross-file validations that are performed.